gflownet/envs/crystals.py

```diff
@@ -263,8 +263,7 @@ def get_parents(self, state=None, done=None, actions=None):
         else:
             parents = []
             actions = []
-            for idx, action in enumerate(self.action_space[:-1]):
-                element, n = action
+            for idx, (element, n) in enumerate(zip(self.action_space[:-1])):
```

Comment:
This breaks the tests, I believe zip is unnecessary.

Reply:
Ugh, it's not that it breaks the tests, the zip is just wrong... Sorry! Fixed.

Reply:
Well, it's both, I was subtly suggesting to use tests before pushing changes, haha.